src/main/java/com/yelp/nrtsearch/server/handler/AddDocumentHandler.java

```diff
@@ -488,7 +564,12 @@ private void updateNestedDocuments(
       }
 
       documents.add(rootDoc);
+      IndexingMetrics.addDocumentRequestsReceived.labelValues(indexName).inc();
+      long timeInMillis = System.currentTimeMillis();
```

Comment:
Sorry, I could have been more clear. I meant doing something like `(ns_end - ns_start) / 1000000.0` so you would record milliseconds as a double with a fractional component.